React-Template-master/src/components/MessageModal.tsx

```diff
@@ -0,0 +1,79 @@
+import React, { useState } from 'react';
+import { Modal, Box, Typography, Button, useTheme } from '@mui/material';
+import { NEUTRAL } from '../theme/palette';
+import { CorrectLogo, WarningLogo } from './Logo';
+import { useTranslation } from 'react-i18next';
+
+interface ConfirmationModalProps {
```

Comment:
The file name and component name doesn't match with the interface name. Make sure you give proper names

Reply:
okay